0_portfolio_input_check_functions.R

```diff
@@ -861,11 +861,19 @@ create_id_columns <- function(portfolio, portfolio_type) {
 }
 
 # FINAL SCRIPTS
+
+get_and_clean_total_fund_list_data <- function() {
+  total_fund_list <- read_rda("data/total_fund_list.rds")
```

Comment:
well i followed the same logic as currency data is processed: 

If new_data == TRUE -> The function get_and_clean_total_fund_list_data is called and takes the rds fromt he /data folder, and in a later function save_cleaned_files the dataframe is saved to save_loc which is the pacta-data folder

if new_data != TRUE -> The fst file from pacta_data is loaded

Reply:
ok, for some reason when I wrote that, I was not seeing `data/total_fund_list.rds` in this PR, so this would fail if `new_data == TRUE`.... but it looks like it is there now.... maybe you changed the name of `data/fund_isin_list.rds`?